src/mozanalysis/metrics/desktop.py

```diff
@@ -1,404 +1,84 @@
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
-from textwrap import dedent
 
-from mozanalysis.metrics import DataSource, Metric, agg_any, agg_sum
+# These metric definitions are deprected.
+# Instead use the metric slug to reference metrics defined
+# in https://github.com/mozilla/jetstream-config
 
-#: DataSource: The clients_daily table.
-clients_daily = DataSource(
-    name="clients_daily",
-    from_expr="mozdata.telemetry.clients_daily",
-)
-
-#: DataSource: The `search_clients_engines_sources_daily`_ table.
-#: This table unpacks search counts from the main ping;
-#: it contains one row per (client_id, submission_date, engine, source).
-#:
-#: .. _`search_clients_engines_sources_daily`: https://docs.telemetry.mozilla.org/
-#:    datasets/search/search_clients_engines_sources_daily/reference.html
-search_clients_engines_sources_daily = DataSource(
-    name="search_clients_engines_sources_daily",
-    from_expr="mozdata.search.search_clients_engines_sources_daily",
-    experiments_column_type=None,
-)
+from mozanalysis.config import ConfigLoader
 
-#: DataSource: A clone of `search_clients_engines_sources_daily`.
-#: Exists for backwards compatibility; new uses should use the new name.
-search_clients_daily = search_clients_engines_sources_daily
+clients_daily = ConfigLoader.get_data_source("clients_daily", "firefox_desktop")
 
-#: DataSource: The main_summary table.
-main_summary = DataSource(
-    name="main_summary", from_expr="mozdata.telemetry.main_summary"
+search_clients_engines_sources_daily = ConfigLoader.get_data_source(
+    "search_clients_engines_sources_daily", "firefox_desktop"
 )
 
-#: DataSource: The events table.
-events = DataSource(
-    name="events",
-    from_expr="mozdata.telemetry.events",
-    experiments_column_type="native",
+search_clients_daily = ConfigLoader.get_data_source(
+    "search_clients_engines_sources_daily", "firefox_desktop"
```

Comment:
Any reason to make another call instead of doing what we had before (below)?
```suggestion
search_clients_daily = search_clients_engines_sources_daily
```

Reply:
Good catch, we can keep what we had before